<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CCloseProjectDialog</class>
 <widget class="QDialog" name="CCloseProjectDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>426</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Close Project Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QPushButton" name="pushButtonSave">
     <property name="text">
      <string>save</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonCancel">
     <property name="text">
      <string>cancel</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButtonNotSave">
     <property name="text">
      <string>not save</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
